bucket/deepstream.json

```
@@ -0,0 +1,28 @@
+{
+    "homepage": "https://deepstream.io",
+    "description": "The open realtime server, a fast and secure data-sync realtime server for mobile, web & iot",
```

No need to mention it is open sourced. This cover license
```suggestion
    "description": "Realtime, fast and secure data-sync server for mobile, web & iot.",
```